docs/vsphere-golden-image.md

```diff
@@ -71,14 +71,15 @@ See [Microsoft documentation](https://docs.microsoft.com/en-us/windows-server/ad
 to install using PowerShell.
 
 Alternatively, the provided PowerShell script [install-openssh.ps1](vsphere_ci/scripts/install-openssh.ps1) aims to 
-programmatically install and configure the OpenSSH Server with the public key file from section [Add the Administrator 
-Public Key](#add-the-administrator-public-key). For example, you can run the following command:
+programmatically install and configure the OpenSSH Server using the public key corresponding to the [private key](../README.md#create-a-private-key-secret)
+provided as part of WMCO setup. For example, you can run the following command:
```

Comment:
I suggest removing, `For example` given you are changing this. This is not an example but rather the only command used. This key is also not provided as part of WMCO setup.

`programmatically install and configure the OpenSSH Server using the public key corresponding to the private key used in the [secret](../README.md#create-a-private-key-secret)`

This is also pointing at the fact that this file needs to be present on the VM, which leads to more [confusion](https://github.com/openshift/windows-machine-config-operator/pull/515#issuecomment-900546406) given SSH is not setup. So maybe we list the commands that they need to run here instead. But yes, they still need to figure out how to get the key over.